### PR TITLE
propagate error in chained jobs

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

**Review by mklkun — APPROVED** [review]

**Review by mboussaa — APPROVED** [review]
LGTM

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1844,6 +1844,20 @@ To provide the _firstName_ and _lastName_ variables parameters to the submitted
 
 The `SubmitJobNoWait` or `SubmitJobAndWait` templates behavior is different in the sense that the latter will wait until the submitted workflow terminates.
 
+In case an error occurs in the chained jobs, a user may want to escalate this error from the submitted (child) job to the parent job by throwing an exception. It is possible to integrate this behavior in the `SubmitJobAndWait` implementation using the <<_scheduler_api,*schedulerapi*>> binding as described in the code below.
+[source,groovy]
+----
+jobid = variables.get("jobID")
+jobState = schedulerapi.getJobState(jobid)
+jobStatus = jobState.getStatus().toString()
+isJobSuccessful = !(jobStatus.equals("FAILED") || jobStatus.equals("CANCELED") || jobStatus.equals("KILLED") || (jobStatus.equals("FINISHED") && jobStatus.getNumberOfFaultyTasks() > 0))
```
Comment (fviale) [inline src/docs/user/ProActiveUserGuide.adoc:1853]: you need to fix this test :-)

Comment (fviale) [reply src/docs/user/ProActiveUserGuide.adoc:1853]: !(jobStatus.equals("FAILED") && !jobStatus.equals("CANCELED") && ...

Comment (aminelouati) [reply src/docs/user/ProActiveUserGuide.adoc]: I applied a fix and double checked this test with real examples.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -1844,6 +1844,20 @@ To provide the _firstName_ and _lastName_ variables parameters to the submitted
 
 The `SubmitJobNoWait` or `SubmitJobAndWait` templates behavior is different in the sense that the latter will wait until the submitted workflow terminates.
 
+In case an error occurs in the chained jobs, a user may want to escalate this error from the submitted (child) job to the parent job by throwing an exception. It is possible to integrate this behavior in the `SubmitJobAndWait` implementation using the <<_scheduler_api,*schedulerapi*>> binding as described in the code below.
+[source,groovy]
+----
+jobid = variables.get("jobID")
+jobState = schedulerapi.getJobState(jobid)
+jobStatus = jobState.getStatus().toString().toUpperCase()
+isJobSuccessful = !(jobStatus.equals("FAILED") || jobStatus.equals("CANCELED") || jobStatus.equals("KILLED") || (jobStatus.equals("FINISHED") && jobState.getNumberOfFaultyTasks() > 0))
```
Comment (fviale) [inline src/docs/user/ProActiveUserGuide.adoc:1853]: (jobStatus.equals("FINISHED") && jobState.getNumberOfFaultyTasks() == 0)

**Review by fviale — APPROVED** [review]

**Pull request merged**